docs/containers-storage.conf.5.md

```diff
@@ -53,12 +53,48 @@ The `storage.options` table supports the following options:
   Maximum size of a container image.   This flag can be used to set quota on the size of container images. (default: 10GB)
 
 **mount_program**=""
-  Specifies the path to a custom program to use instead for mounting the file system.
+  Specifies the path to a custom program to use instead of using kernel defaults for mounting the file system.
+
+      mount_program = "/usr/bin/fuse-overlayfs"
 
 **mountopt**=""
 
   Comma separated list of default options to be used to mount container images.  Suggested value "nodev".
 
+**remap-uids=**""
+**remap-gids=**""
+
+  Remap-UIDs/GIDs is the mapping from UIDs/GIDs as they should appear inside of
+a container, to UIDs/GIDs as they should appear outside of the container, and
+the length of the range of UIDs/GIDs.  Additional mapped sets can be listed
+and will be heeded by libraries, but there are limits to the number of
+mappings which the kernel will allow when you later attempt to run a
+container.
+
+     remap-gids = 0:1668442479:65536
+     remap-uids = 0:1668442479:65536
+
```

Comment:
Should these numbers have any correspondence to numbers in any file on the host machine?  If so, or if not, you might want to say so.

Reply:
Nope they are just UIDs.  Theoretically they should be allocated in the /etc/subuid file.